tests/cypress/integration/issue_1919_check_text_attr.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check label attribute changes', () => {
+
+    const issueId='1919'
+    const labelName=`Issue ${issueId}`
+    const taskName=`New annotation task for ${labelName}`
+    const attrName=`Attr for ${labelName}`
+    const textDefaultValue='Some default value for type Text'
+    const image=`image_${issueId}.png`
+    const newLabelAttrValue = 'New attribute value'
+    const width='800'
+    const height='800'
+    const posX=10
+    const posY=10
+    const color='gray'
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.get('[type="submit"]').click()
```

Comment:
Shouldn't ``cy.get('[type="submit"]').click()`` be a part of ``cy.login()``?

Reply:
Yes it's better that way. Moved.